File: Backend/JQZHomeCareProject/JQZHomeCareProject.Domain/Entities/Rating.cs
using System;
using System.Collections.Generic;
using System.Text;
using JQZHomeCareProject.Domain.Common;

namespace JQZHomeCareProject.Domain.Entities
{
    public class Rating : BaseEntity
    {
        public Guid PractitionerId { get; set; }
        public Practitioner? Practitioner { get; set; }
        public DateTime Month { get; set; }
        public int Score { get; set; }
        public string? Comments { get; set; }
    }
}

File: jqz-homecare-admin/src/app/core/services/rating.ts
export interface Rating {
  id: string;

  practitionerId: string;

  month: number;

  score: number;

  comments?: string | null;
}

export interface CreateRating {
  month: number;

  score: number;

  comments?: string | null;
}

export interface UpdateRating {
  score: number;

  comments?: string | null;
}
